packages/firebase_remote_config/firebase_remote_config/test/firebase_remote_config_test.dart

```diff
@@ -19,90 +17,85 @@ MockFirebaseRemoteConfig mockRemoteConfigPlatform = MockFirebaseRemoteConfig();
 void main() {
   setupFirebaseRemoteConfigMocks();
 
-  RemoteConfig remoteConfig;
-
-  DateTime mockLastFetchTime;
-  RemoteConfigFetchStatus mockLastFetchStatus;
-  RemoteConfigSettings mockRemoteConfigSettings;
-  Map<String, RemoteConfigValue> mockParameters;
-  Map<String, dynamic> mockDefaultParameters;
-  RemoteConfigValue mockRemoteConfigValue;
+  late RemoteConfig remoteConfig;
+  DateTime mockLastFetchTime = DateTime(2020);
+  RemoteConfigFetchStatus? mockLastFetchStatus =
+      RemoteConfigFetchStatus.noFetchYet;
+  RemoteConfigSettings? mockRemoteConfigSettings = RemoteConfigSettings(
+    fetchTimeout: const Duration(seconds: 10),
+    minimumFetchInterval: const Duration(hours: 1),
+  );
+  Map<String, RemoteConfigValue> mockParameters = <String, RemoteConfigValue>{};
+  Map<String, dynamic> mockDefaultParameters = <String, dynamic>{};
+  RemoteConfigValue mockRemoteConfigValue = RemoteConfigValue(
+    <int>[],
+    ValueSource.valueStatic,
+  );
 
   group('$RemoteConfig', () {
-    FirebaseRemoteConfigPlatform.instance = mockRemoteConfigPlatform;
-
     setUpAll(() async {
```

Comment:
Why did the setUpAll change (such as the `when` being removed from it)?

Reply:
for better separation of tests